test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeUnsafeTests.java

```diff
@@ -303,9 +292,6 @@ static public void testGetValueOnVTWithLongFields() throws Throwable {
 		if (isFlatteningEnabled) {
 			assertEquals(x.l, vtLongPoint.x.l);
 			assertEquals(y.l, vtLongPoint.y.l);
-		} else {
-			assertEquals(x.l, myUnsafe.getLong(vtLongPoint, vtLongPointOffsetX));
-			assertEquals(y.l, myUnsafe.getLong(vtLongPoint, vtLongPointOffsetY));
 		}
 
 		assertEquals(vtLongPoint.x.l, 123);
```

Comment:
good point, I'll work on this now

Reply:
I updated `IntWrapper` in e2d23e0 to contain a `ValueTypeInt` instead of an `int`. I didn't create a `NonVTPoint2D` because I am already using a very similar `Point2D` class for other tests and I didn't want to get them confused